src/teleop/download_map/download_map.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+import zipfile
+import json
+import boto3
+import subprocess
+
+
+# If System has issues with number of file watcher when using GUI with downloaded map
+# Run "sudo sysctl -w fs.inotify.max_user_watches=100000"
+def main():
+    result = subprocess.Popen(
+        ["rospack", "find", "mrover"], stdout=subprocess.PIPE, stderr=subprocess.PIPE
+    )  # runs rospack find mrover to get directory
+    m_bytes, err = result.communicate()
```

Comment:
Is it worth checking err for failure? If not, it may be good to add a comment explaining why